addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/handler/DeviceHandler.java

```diff
@@ -212,12 +304,13 @@ public void addDeviceList(DeviceModel model) {
     }
 
     /**
-     * Builds a {@link ThingUID} from a device model. The UID is build from
-     * the {@link BindingConstants#BINDING_ID} and value of
+     * Builds a {@link ThingUID} from a device model. The UID is build from the
+     * {@link BindingConstants#BINDING_ID} and value of
      * {@link DeviceModel#getProductName()} in which all characters NOT matching
      * the regex [^a-zA-Z0-9_] are replaced by "_".
      *
-     * @param device Discovered device model
+     * @param device
```

Comment:
Hey Martin, I am feeling like this is the 1000 point where my code formatter did not deliver the code format for contributions which is needed. I installed my IDE like decribed in the docs and use the built-in code formatter. What is going wrong?

As a side note: In eclipse it is possible to export and import code formatter profiles. Maybe you can provide me a working import file for the openHAB code formatter profile. Maybe we (openHAB community) can provide it as download in the documentation. What do you think?